projects/extension/sql/idempotent/013-vectorizer-api.sql

```diff
@@ -31,6 +29,9 @@ create or replace function ai.create_vectorizer
 , queue_table pg_catalog.name default null
 , grant_to pg_catalog.name[] default ai.grant_to()
 , enqueue_existing pg_catalog.bool default true
+, embedding_batch_schema pg_catalog.name default null
+, embedding_batch_table pg_catalog.name default null
+, embedding_batch_chunks_table pg_catalog.name default null
```

Comment:
Let's move this arguments to 

https://github.com/timescale/pgai/blob/59c32947972bea851193a6a382006c169eed1969/projects/extension/sql/idempotent/008-embedding.sql#L4-L22

Let's also add another attribute to the  json returned by `ai.embedding_openai` called `use_batch_api` that'll take a boolean.

That way the batch logic can be contain in the openAI embedding, since it's the only one that uses it. Users will enable it like:

```sql
select ai.create_vectorizer(
    'public.blogs'::regclass
  , embedding=>ai.embedding_openai(
      'text-embedding-3-small', 
      1536, 
      api_key_name=>'OPENAI_API_KEY', 
      use_batch_api=>true)
  , chunking=>ai.chunking_recursive_character_text_splitter('content')
  , formatting=>ai.formatting_python_template('title: $title url: $url content: $chunk')
);
```

Reply:
Done.